### PR TITLE
Revert "[RFC] Reference global classes via `use`" (re-vote)

### PR DESCRIPTION
**Pull request by Ocramius**
Reverts doctrine/coding-standard#43 - this is just to allow discussing whether this rule should be part of doctrine/coding-standard or not.

 * If you want it to stay, "request changes"
 * If you want it to go away, "approve"

### CODE REVIEW

File: lib/Doctrine/ruleset.xml [hunk 1]
```diff
@@ -168,7 +168,7 @@
         <properties>
             <property name="allowFallbackGlobalConstants" type="boolean" value="false"/>
             <property name="allowFallbackGlobalFunctions" type="boolean" value="false"/>
-            <property name="allowFullyQualifiedGlobalClasses" type="boolean" value="false"/>
+            <property name="allowFullyQualifiedGlobalClasses" type="boolean" value="true"/>
```
Comment (deeky666) [inline lib/Doctrine/ruleset.xml]: As an alternative I could live with treating global constants and classes the same (either way).

**Comment by kimhemsoe** [comment]
I think it should stay in.

**Comment by Ocramius** [comment]
I think this should stay in (can't put my `request changes` as I'm the PR author)

File: lib/Doctrine/ruleset.xml (hunk 1 above)
Comment (Ocramius) [reply lib/Doctrine/ruleset.xml]: Global functions/constants is not really viable as it creates an absolutely mind-boggling amount of inline diffs.

**Comment by Ocramius** [comment]
@Majkl578 I'd likely close this one, given the current state of reviews

**Comment by Majkl578** [comment]
I am okay to keep this for consistency, but would prefer discussion first, not after. :+1:

**Comment by Ocramius** [comment]
@Majkl578 absolutely: let's see how the 4 approvals approach works for us.

**Pull request closed without merging**